packages/core-ethereum/src/indexer/index.ts

```diff
@@ -516,6 +521,27 @@ class Indexer extends (EventEmitter as new () => IndexerEventEmitter) {
 
     await this.processUnconfirmedEvents(blockNumber, lastDatabaseSnapshot, blocking)
 
+    // resend queuing transactions
+    if (
+      this.chain.getAllQueuingTransactionRequests().length > 0 &&
+      (await this.chain.getNativeBalance(this.address)).toBN().gt(new BN(0))
```

Comment:
This creates another RPC call per `block` event. We should rather use the cached value.